src/test/java/CodeCheckTest.java

```diff
@@ -17,16 +17,19 @@ public class CodeCheckTest {
     private final String testConfPath = FileSystems.getDefault()
             .getPath("").toAbsolutePath() + File.separator + "test-config.properties";
 
-    private final String resultFolder = "test-results";
+    protected static final String resultFolder = "test-results";
     private final String resultNamePrefix = "test-result_{nr}";
     private final String resultFileName = "test-result_0.txt";
 
     @Test
     public void executeTest() throws Exception {
-        delete(FileSystems.getDefault()
-                .getPath("").toAbsolutePath() + File.separator + resultFolder);
+        delete(FileSystems.getDefault().getPath("").toAbsolutePath() + File.separator + resultFolder);
 
-        Util.createFile(testConfPath, true);
+        Log.debug("#--- First: " + FileSystems.getDefault().getPath("").toAbsolutePath() + File.separator + resultFolder);
```

Comment:
only one would suffice

Reply:
This particular case was fixed in another commit. But I get what you mean.